src/dynamics/circle/rotation_number/translation_number.lean

```diff
@@ -652,4 +731,80 @@ begin
   exact (f^n).translation_number_eq_int_iff (f.continuous_pow hf n)
 end
 
+/-- Consider two actions `f₁ f₂ : G →* circle_deg1_lift` of a group on the real line by lifts of
+orientation preserving circle homeomorphisms. Suppose that for each `g : G` the homeomorphisms
+`f₁ g` and `f₂ g` have equal rotation numbers. Then there exists a lift of a circle homeomorphism
+`F` such that `F ∘ f₁ g = f₂ g ∘ F` for all `g : G`.
```

Comment:
`F` is not a homeomorphism in general! The `f_i g` are, because you are considering a group action so they are invertible so they are continuous, but the continuity is not obvious just from the statement, so it's maybe worth mentioning this in the docstring.

Reply:
I wonder how have I made a typo like this. Of course, it is not a homeomorphism!